CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 17.2.1 (01/22/2025)
```

Comment:
Let's call out https://github.com/gravitational/teleport/pull/51085 as a security fix.

Reply:
Done. Is that fine as a separate section like that, just listing the changelog entry, or is there some more wording to add around this?

Reply:
Yeah this is fine. And let's mark the Github release a security release.

Reply:
Will do. I've already updated the release checklist for the step where the release is created to add a reminder and how-to for security releases.